examples/tblind.js

```diff
@@ -0,0 +1,66 @@
+// Example of how threshold signing is expected to be consumed from the JS side
+
+// Import the library
+const threshold = require("../pkg/threshold")
+const crypto = require('crypto')
+
+// Get a message and a secret for the user
+const msg = Buffer.from("hello world")
+const user_seed = crypto.randomBytes(32)
+
+// Blind the message
+const blinded_msg = threshold.blind(msg, user_seed)
+const blind_msg = blinded_msg.message
+
+// Generate the secret shares for a 3-of-4 threshold scheme
+const t = 3;
+const n = 4;
+const keys = threshold.threshold_keygen(n, t, crypto.randomBytes(32))
+const shares = keys.shares
+const polynomial = keys.polynomial
+
+// each of these shares proceed to sign teh blinded sig
+let sigs = []
+for (let i = 0 ; i < keys.num_shares(); i++ ) {
+    const sig = threshold.partial_sign(keys.get_share(i), blind_msg)
+    sigs.push(sig)
+}
+
+// The combiner will verify all the individual partial signatures... 
```

Comment:
This part is commented out as it's blocked by https://github.com/celo-org/celo-threshold-bls-rs/pull/9

Reply:
Fixed by https://github.com/celo-org/celo-threshold-bls-rs/pull/9/commits/857465099ef2c06f83a879b8f8dd4bacf8e0fb6e. Partial signing and verification should not map to curve.